code/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -40,14 +41,20 @@
   @Inject
   private RobotContainer(final ILogger logger, IDrivetrainSubsystem drivetrain,
       DrivetrainControllerCommand driveCommand, IIntakeSubsystem intake, DriveArmCommand driveArmCommand,
-      IClimbSubsystem climb, ClimbControllerCommand climbCommand, Provider<Easy23StrategyCommand> autonomousCommand) {
+      IClimbSubsystem climb, ClimbControllerCommand climbCommand, Provider<Easy23StrategyCommand> autonomousCommand,
+      ILedSubsystem ledSubsystem, SetAllianceColorCommand setAllianceColorCommand) {
     super(logger);
     _autonomousCommand = autonomousCommand;
 
     drivetrain.setDefaultCommand(driveCommand);
     intake.setDefaultCommand(driveArmCommand);
     climb.setDefaultCommand(climbCommand);
 
+    System.out.println(ledSubsystem);
```

Comment:
All of these debug prints should be removed now

Reply:
removed debug